### DISCUSSION TITLE
¿Cómo copio las respuestas (o mensajes) de un formulario a un Bot?

### DISCUSSION THREAD

**Discussion by ValeMP** [Cotalker/documentation#39] [2. Questions and Answers] [answered]
Hola!

Para poder copiar las respuesta de un formulario (o mensajes) configurado en un bot **triggered by a survey**, se debía editar un parámetro del formulario o bot o otro elemento (no recuerdo muy bien). 

Alguien me podría aclarar, por favor, ¿Qué parámetro y qué valor debe tener para poder hacer uso del bot de copiar formulario? 

Gracias!

**Comment by acedward** [comment, marked as answer]
@ValeMP Es posible obtener los mensajes originales como contexto del bot activando la opción "save from response"

Quedan disponibles en el campo { ..., messages }, donde messages es un arreglo de COTMessages 
y para acceder a ellos $VALUE#messages 

![image](https://user-images.githubusercontent.com/3951895/113764092-8271b080-96e8-11eb-9ee1-736620bb689d.png)

@guillevalin Tenemos que mejorar el nombre de este input.
@Mauriceac Debemos que actualizar la documentación con este input.

**Reply by ValeMP** [reply]
aaah para eso servia el boton! ME daba una idea de que se guardaba para mostrarlo en la seccion de reportes! 
Gracias!
